### PR TITLE
disable file imports when running on external server

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request disables "From File..." options for importing catalogs and footprints from a file in cases where jdaviz is running on an external server.

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone. Bugfix milestone also needs an accompanying backport label.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Comment by kecnry** [comment]
NOTE: I'm not sure if the auto backport will still work for 3.10.x, if it does not, we can manually backport there (probably will be a changelog conflict anyways due to the 4.0.x changelog entry not existing).

File: CHANGES.rst [hunk 1]
```diff
@@ -76,6 +76,8 @@ Cubeviz
 Imviz
 ^^^^^
 
+- Remove "From File.." option when running on an external server. [#3239]
```
Comment (pllim) [inline CHANGES.rst:79]: Actually do we want to advertise this?

Comment (kecnry) [reply CHANGES.rst:79]: it'll be in the PR history regardless 🤷‍♂️

Comment (pllim) [reply CHANGES.rst:79]: If you insist on change log, I think also need entry under 4.0.x section.

**Comment by pllim** [comment]
p.s. Are these the only ones we need to block? Any others?

File: CHANGES.rst (hunk 1 above)
Comment (kecnry) [reply CHANGES.rst:79]: there is (this one you're commenting on is 4.0.1, the other one is 3.10.4)

**Comment by kecnry** [comment]
these are all.  @gibsongreen can follow the same pattern when implementing subset importing.

**Review by pllim — APPROVED** [review]
Okie dokie then. Feel free to merge. Thanks for the quick fix!

cc @havok2063

**Review by gibsongreen — APPROVED** [review]
Changes to the plugins look good to me and I have this PR bookmarked for the future effort!

**Pull request merged**